packages/core/src/__testfixtures__/transformtest.input.ts

```diff
@@ -1,90 +1,90 @@
 export const objDict = {
```

Comment:
This could be the reason for CI failure. Consider reverting the changes in both files in this folder, unless they're intentional?

Reply:
I reverted both files to Stella's original commit using this command: `git checkout db17b278bebe04f70d8e8c8135189d403f67aa2e -- transformtest.output.ts` and added them to the commit 933ac6ec62fdc6f4bbfab854f9d2aabfc55d0359.

The tests also all pass locally.

However, after the pre-commit hook, it appears that there are no changes made. My guess is that the pre-commit hook has been auto-formatting these files and adding whitespace that may interfere with the tests (though I'm not sure why the tests would work locally and not on CircleCI).

I could try to turn off the hook, but probably it would auto-format everyone else's commits for these files as well.

How should I proceed here?

Reply:
Yep that sounds about right. Just add this folder to `prettierignore`

Reply:
Looks like it works in CircleCI now.